packages/grid/src/ThemeContext.ts

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import GridTheme, { GridTheme as GridThemeType } from './GridTheme';
+
+export const ThemeContext: React.Context<
+  Partial<GridThemeType>
+> = React.createContext(GridTheme as Partial<GridThemeType>);
```

Comment:
Change the default to an empty object, since we merge the value with `GridTheme` in `Grid.getTheme` anyway.